narwhals/series.py

```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this Series are in the other Series.
+
+        Arguments:
+            other: Series or sequence of primitive type.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s_pd = pd.Series([1, 2, 3])
+            >>> s_pl = pl.Series([1, 2, 3])
+
+            We define a library agnostic function:
+
+            >>> def func(s_any):
+            ...     s = nw.from_native(s_any, series_only=True)
+            ...     s = s.is_in(pl.Series([3, 2, 8]))
```

Comment:
we shouldn't use anything polars-specific in these examples, can you just use `is_in([3, 2, 8])`?

Reply:
done